app/components/avo/divider_component.rb

```diff
@@ -1,9 +1,6 @@
 # frozen_string_literal: true
 
-class Avo::DividerComponent < ViewComponent::Base
-  attr_reader :label
-
-  def initialize(label = nil, **args)
-    @label = label
-  end
+class Avo::DividerComponent < Avo::BaseComponent
+  prop :label, _Nilable(String), :positional, reader: :public
+  prop :args, Hash, :**
```

Comment:
I don’t think this is needed, but it was allowed before so didn’t want to break anything.